CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Pathfinder now fetches data concurrently from the feeder gateway when catching up. The `--gateway.fetch-concurrency` CLI option can be used to limit how many blocks are fetched concurrently (the default is 8).
 
+### Changed
+
+- Ethereum RPC API now favors Websocket endpoints over HTTP.
```

Comment:
Well, it's not quite "favors" rather "requires" Websocket?
I think we should also explicitly mention here that the configuration should be changed to use a Websocket URL and that in case an HTTP/HTTPS URL is specified we try to convert that to WS/WSS?